client/extensions/woocommerce/woocommerce-services/views/packages/style.scss

```
@@ -37,7 +38,44 @@
 
 	@include breakpoint( '>660px' ) {
 		width: 610px;
-		height: 530px;
+		height: auto;
+		padding: 0;
+
+		.dialog__content {
+			padding: 0;
+
+			.packages__add-edit-title {
+				padding: 22px;
+				margin: 0;
+				border-bottom: 1px solid #EDEFF0;
+				font-size: 20px;
+			}
+
+			.packages__mode-select {
+
+				.segmented-control__link {
+					border-radius: 0;
+					border: 0 none;
+					padding: 15px 0;
+				}
+
```

```suggestion
.segmented-control__link:focus {
    box-shadow: none;
}
```
This will fix the blue highlight around 3 of the borders when you first click on a link.

---

:+1: done!